docs/src/main/asciidoc/adapters/gcp-intro.adoc

```diff
@@ -0,0 +1,119 @@
+:branch: master
+
+=== Google Cloud Functions (Alpha)
+
+The Google Cloud Functions adapter enables Spring Cloud Function apps to run on the https://cloud.google.com/functions[Google Cloud Functions] serverless platform.
+You can either run the function locally using the open source https://github.com/GoogleCloudPlatform/functions-framework-java[Google Functions Framework for Java] or on GCP.
+
+==== Getting Started
+
+Let’s start with a simple Spring Cloud Function example:
+
+[source, java]
+----
+@SpringBootApplication
+public class CloudFunctionMain {
+
+	public static void main(String[] args) {
+		SpringApplication.run(CloudFunctionMain.class, args);
+	}
+
+	@Bean
+	public Function<String, String> uppercase() {
+		return value -> value.toUpperCase();
+	}
+}
+----
+
+===== Test locally
+
+Start by adding the Maven plugin provided as part of the Google Functions Framework for Java.
+
+[source, xml]
+----
+<plugin>
+    <groupId>com.google.cloud.functions</groupId>
+    <artifactId>function-maven-plugin</artifactId>
+    <version>0.9.1</version>
+    <configuration>
+        <functionTarget>org.springframework.cloud.function.adapter.gcloud.GcfSpringBootHttpRequestHandler</functionTarget>
+        <port>8080</port>
```

Comment:
I think the port can be omitted.

Reply:
I know, but I wanted to show it here so that people know they can change it.